azurelinuxagent/agent.py

```diff
@@ -63,6 +63,9 @@ def __init__(self, verbose, conf_file_path=None):
                                  path="/var/log/waagent.log")
         logger.add_logger_appender(logger.AppenderType.CONSOLE, level,
                                  path="/dev/console")
+        logger.add_logger_appender(logger.AppenderType.TELEMETRY,
+                                   logger.LogLevel.WARNING,
+                                   path=event.add_log_event)
```

Comment:
This is a little bit of a hack.  I am open to ideas.

I overload the path parameter to pass a lambda that actually does the logging. I did this because this method calls 3 more methods all with the same parameters so the appropriate log appender is added.  I could have created appropriate overloads, but this *seemed* better.

I also use a lambda because I didn't want to create a situation where logger.py included event.py, which already includes logger.py.